### ISSUE TITLE
svg is not interactive

### ISSUE THREAD

**Issue by dragonly** [baverman/flameprof#8]
small bars are invisible, and cannot be seen even zoomed in.

**Comment by notpushkin** [comment]
Try running it through flamegraph.pl, as in the last example in the README:

```sh
flameprof --format=log requests.prof | flamegraph > requests-flamegraph.svg
```

This results in an interactive chart with click-to-zoom capabilities, like [this one](https://camo.githubusercontent.com/3cf8b038a93e8c88870c10a3169f35ee3bc959100cedd2ecc553f758aab8c084/68747470733a2f2f63646e2e7261776769742e636f6d2f62617665726d616e2f666c616d6570726f662f6d61737465722f696d672f72657175657374732d666c616d6567726170682e737667).

**Comment by con-f-use** [comment]
Btw. if you want something interactive, try [snakeviz](https://jiffyclub.github.io/snakeviz/).

**Comment by MFShams** [comment]
The graph was not interactive when I used the my Linux image viewer.
However, I ran the same file in Edge browser and it became interactive (I tested this on a Windows operating system)

**Comment by notpushkin** [comment]
@MFShams Yep, image viewers generally don't evaluate JS which in this case is required for interactivity. Any browser should be okay though.
